### PR TITLE
#8640 Add systemd configuration file

### PR DESCRIPTION
**Pull request by blcksrx**
### CATEGORY

Choose one

- [ ] Bug Fix
- [X] Enhancement (new features, refinement)
- [ ] Refactor
- [ ] Add tests
- [ ] Build / Development Environment
- [ ] Documentation

### SUMMARY
Add systemd configuration file in repo refer to #8640 

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: contrib/systemd/superset-webserver.service [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+[Unit]
+Description =Superset Webserver Daemon
+After =network.target postgresql.service mysql.service redis.service rabbitmq-server.service
+Wants =postgresql.service mysql.service redis.service rabbitmq-server.service
```
Comment (dpgaspar) [inline contrib/systemd/superset-webserver.service]: Using `After` and `Want` do not impose hard requisites, yet they assume everything is deployed on the same instance (a really bad idea)

File: contrib/systemd/superset-webserver.service [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+[Service]
+EnvironmentFile=/etc/sysconfig/superset
+User =superset
+Group =superset
+Type =simple
+ExecStart =/bin/gunicorn -w 10 -k gevent -t 120 -b  0.0.0.0:6666 --limit-request-line 0 --limit-request-field_size 0 --statsd-host localhost:8125 superset:app
```
Comment (dpgaspar) [inline contrib/systemd/superset-webserver.service]: These values are hard to change, consider `EnvironmentFile` use or assuming/creating a `gunicorn.conf`

File: contrib/systemd/superset-webserver.service [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+After =network.target postgresql.service mysql.service redis.service rabbitmq-server.service
+Wants =postgresql.service mysql.service redis.service rabbitmq-server.service
+
+[Service]
+EnvironmentFile=/etc/sysconfig/superset
+User =superset
+Group =superset
+Type =simple
+ExecStart =/bin/gunicorn -w 10 -k gevent -t 120 -b  0.0.0.0:6666 --limit-request-line 0 --limit-request-field_size 0 --statsd-host localhost:8125 superset:app 
+Restart =on-failure
```
Comment (dpgaspar) [inline contrib/systemd/superset-webserver.service]: Should this be more like: https://docs.gunicorn.org/en/stable/deploy.html#systemd

File: contrib/systemd/superset-webserver.service (hunk 1 above)
Comment (blcksrx) [reply contrib/systemd/superset-webserver.service]: I tried to created systemd files like `apache airflow`. Okay, you're right, I will modify it as soon as possible.

File: contrib/systemd/superset-webserver.service (hunk 2 above)
Comment (blcksrx) [reply contrib/systemd/superset-webserver.service]: I will.

File: contrib/systemd/superset-webserver.service (hunk 3 above)
Comment (blcksrx) [reply contrib/systemd/superset-webserver.service]: Thanks.

**Pull request closed without merging**